doc/contributing.md

```diff
@@ -0,0 +1,125 @@
+
+We drive the bot by tagging issues with specific labels which cause the bot engine to close issues, merge branches, etc. This bot engine helps us keep the repo clean by automating the process of notifying appropriate parties if/when information/follow-up is needed, and closing stale issues/PRs after reminders have remained unanswered for several days.
+
+Therefore, If you do file issues, or create PRs, please keep an eye on your GitHub notifications. If you do not respond to requests for information, your issues/PRs may be closed automatically.
+
+## Reporting an bug/issue
+
+### DO NOT post "+1" comments
+
+⚠ DO NOT post "+1", "me too", or similar comments: If you don't have any additional info/context to add but would like to indicate that you're affected by the issue, upvote the original issue by clicking its [+😊] button and hitting 👍 (+1) icon. This way we can actually measure how impactful an issue is.
+
+Search existing issues
+Be sure to search existing open and closed issues before filing a new issue: Nobody enjoys reading 20 reports of the same issue - we'd MUCH prefer it if you add comments to an existing issue describing the same/similar issue.
+
+This project is moving fast and its likely someone else has found the problem you're seeing, and someone may be working on or have already contributed a fix!
+
+If not, great - please file a new issue:
+
+### File a new Issue
+
+When you hit "New Issue", select the type of issue closest to what you want to report/ask/request:
+!(/docs/images/new-issue-template.png)
+
+**Complete the requested information and provide as much information as possible**. The more information you provide, the more likely your issue/ask will be understood and fixed. Wherever possible, provide steps to reproduce the issue: **Repro steps are among the most valuable part of a bug report!**
+
+Other Helpful information includes:
+
+* What device you're running (inc. CPU type, memory, disk, etc.)
+* What build of Windows your device is running
+  
+  👉 Tip: run `ver` in PowerShell/Cmd, e.g.:
+
+    ```powershell
+    C:\> ver
+
+    Microsoft Windows [Version 10.0.18900.1001]
+    ```
+
+* What tools and apps you're using (e.g. VS 2019, VSCode, etc.)
+* Don't assume we're experts in setting up YOUR environment and don't assume we are experts in `<your distro/tool of choice>`. Teach us to help you!
+* What steps do we need to take to reproduce the issue? Assume we love to read repro steps. As much detail as you can stand is probably _barely_ enough detail for us!
+* If you're reporting a particular character/glyph not rendering correctly, the specific Unicode codepoint would be MOST welcome
+* Prefer error message text where possible, or screenshots of errors if text cannot be captured
+
+👉Tip: We MUCH prefer text command-line script than screenshots of command-line script.
+

```

Comment:
Something to consider - Often times when someone files a new issue they want to work on it.   Maybe we can add some simple guidance around _requesting to be assigned the issue_ .   The helps prevent duplicate work.   

If they don't request the issue be assigned to them then we can assume it is a "help wanted" issue.  

The comments below around  **discussing the issue/feature/ask with the team before starting any work**  still apply.

Reply:
I agree that we should clarify this point. We are assuming new issues in the tracker are automatically our problem unless someone from the community explicitly asks to take ownership of it.

Reply:
Added clarification

Reply:
I don't like the clarification. It's more of "indicate if you want to do it yourself because by default issues filed are considered the responsibility of the core contributors unless marked `Help-Wanted` during a triage meeting."

Reply:
Improved.